src/App.js

```diff
@@ -1,7 +1,45 @@
 import './App.css';
+import Main from './views/Main/Main';
+import { BrowserRouter, Switch, Route, NavLink } from 'react-router-dom';
+import Films from './views/Films/films';
+import Books from './views/Books/books';
+import Characters from './views/Characters/characters';
 
 function App() {
-  return <div className="App"></div>;
+  // console.log('films', films);
+  // console.log('books', books);
+
+  return (
+    <BrowserRouter>
+      <div className="App">
+        <div></div>
+        <NavLink to="/books">Books</NavLink>
+        <NavLink to="/characters">Characters</NavLink>
+        <NavLink to="/films">Films</NavLink>
+        <NavLink to="/">Main</NavLink>
```

Comment:
Be sure to add these links into a header component and render them above the switch component.